packages/swap/src/quoting/schemas.ts

```diff
@@ -6,21 +6,21 @@ export const marketMakerResponseSchema = z.union([
     .object({
       id: z.string(),
       intermediate_amount: numericString,
-      egress_amount: numericString,
+      output_amount: numericString,
```

Comment:
i kind of unintentionally changed this and then figured that the old name was misleading since we have an egress fee. this should be fine as none of the market makers is connected to this system anyway